### PR TITLE
Closes #161

### LINKED ISSUE #161: Pattern Library Downloaded on Install
using npm install

### LINKED ISSUE #166: Import SCS from repo
wget ____ -O /path

### PR DESCRIPTION
**Pull request by trevren11**
Downloads a copy of the current pattern library on install

### CODE REVIEW

File: core/install_scss.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var Download = require('download');
```
Comment (webnard) [inline core/install_scss.js]: Will this run automatically when we call `npm install` or does something else need to happen first?

Comment (trevren11) [inline core/install_scss.js]: Yes, it happens with npm install.

Comment (webnard) [inline core/install_scss.js]: It doesn't seem to be working on my system. Have you added something to your `package.json` file that didn't get pushed up?

Comment (trevren11) [inline core/install_scss.js]: I did add it to the package.json but it wasn't added. I just added in though

Comment (trevren11) [inline core/install_scss.js]: Ahh it left out something else too, where it installs it, slightly important. I will make a new branch and request. Oi

Comment (webnard) [inline core/install_scss.js]: Haha, okay. Let me know when it's up

On Tue, Mar 17, 2015 at 9:24 AM, trevren11 <notifications@github.com> wrote:

> In core/install_scss.js
> <https://github.com/BYU-ODH/hlrdesk/pull/181#discussion_r26582459>:
>
> > @@ -0,0 +1,10 @@
> > +var Download = require('download');
>
> Ahh it left out something else too, where it installs it, slightly
> important. I will make a new branch and request. Oi
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BYU-ODH/hlrdesk/pull/181/files#r26582459>.
>



-- 
 ian hunter

**Pull request closed without merging**